git-java/textconv-javap

```diff
@@ -6,6 +6,7 @@ javapBin=javap
 [[ -x "$javapBin" ]] || javapBin="$JAVA_HOME/bin/javap"
 [[ -x "$javapBin" ]] || javapBin="/etc/alternatives/java_sdk/bin/javap"
 [[ -x "$javapBin" ]] || javapBin="/etc/alternatives/javap"
+[[ -x "$javapBin" ]] || javapBin="$HOME/.jenv/shims/javap"
```

Comment:
uhh.. that's just the diff format, click "View" and you'll see what it looks like.